### PR TITLE
RHDEVDOCS-3904: Adding a GitHub repository containing pipelines

### PR DESCRIPTION
**Pull request by shipsing**
Purpose: To resolve this issue: https://issues.redhat.com/browse/RHDEVDOCS-3904

Aligned team: DevConsole

OCP version this PR applies to: 4.11

Content for preview:

SME review completed: @karthikjeeyar

QE review completed: @sanketpathak 

Peer review completed: @jc-berger

### CODE REVIEW

**Comment by shipsing** [comment]
This is the preview of the newly added content:

Add-a-git-repository1

Adding-a-git-repository2

**Comment by shipsing** [comment]
Hello @karthikjeeyar, Could you please help with SME review of this PR?
Thanks

File: modules/odc-adding-a-GitHub-repository-containing-pipelines.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+====
+
+.Prerequisites
+* Ensure that your cluster administrator has configured the required GitHub applications in the administrator perspective.
+
+.Procedure
+. In the developer perspective, choose the namespace or project in which you want to add your GitHub repository.
+. Navigate to *Pipelines* using the left navigation pane.
+. Click *Create* -> *Repository* on the right side of the Pipelines page.
+. Select your *Git Repo URL* from the dropdown list and the console automatically fetches the repository name.
```
[the remark this answers is not in the document]
Comment (shipsing) [reply modules/odc-adding-a-GitHub-repository-containing-pipelines.adoc]: @karthikjeeyar, Incorporate this comment. Please verify and confirm.
Thanks

**Comment by shipsing** [comment]
Hello @sanketpathak, Could you please help with QE review of this PR?
Thanks

**Comment by sanketpathak** [comment]
LGTM

**Review by jc-berger — APPROVED** [review]
/lgtm

consulted the openshift checklist when conducting this peer review: https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.mgz8v4w1kg86

**Pull request merged**